### PR TITLE
[gpctl] Support multiple container for debug logs

### PR DESCRIPTION
**Pull request by csweichel**
## Description
This PR makes `gpctl debug log` support multiple containers so that it works e.g. for `ws-daemon`.

## How to test
```
gpctl debug log ws-dameon
```

## Release Notes

```release-note
`gpctl debug log` supports multiple containers
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6542#issuecomment-959251434" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/6542#issuecomment-959251434" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: dev/gpctl/cmd/debug-log.go [hunk 1]
```diff
@@ -73,6 +70,29 @@ var debugLogCmd = &cobra.Command{
...
+	readychan, errchan := util.ForwardPort(ctx, cfg, namespace, podName, port)
+	select {
+	case <-readychan:
+	case err := <-errchan:
+		return err
+	case <-ctx.Done():
+		return ctx.Err()
+	}
+
+	_, err = http.Post(fmt.Sprintf("http://localhost:%d/debug/logging", freePort), "", bytes.NewReader([]byte(`{"level":"debug"}`)))
```
Comment (geropl) [inline dev/gpctl/cmd/debug-log.go:92]: If we add `Content-Type: application/json` this will also work for TS components. :+1:

Comment (csweichel) [reply dev/gpctl/cmd/debug-log.go:92]: Will add

Comment (csweichel) [reply dev/gpctl/cmd/debug-log.go]: Done

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: d004c31dac0c54d2690de3cf60df172c69928b23</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6542#issuecomment-960870801" title="LGTM">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6542#issuecomment-959251434" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/6542#issuecomment-959251434" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf,csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**